tests/test_task.py

```diff
@@ -33,13 +33,6 @@ def test_missing_artifacts(responses, create_task):
         status=404,
```

Comment:
Could you remove the `# First we'll check the new deployment.` comment, since it no longer applies?